ethereum/core/src/jmh/java/org/hyperledger/besu/ethereum/vm/operations/CountLeadingZerosOperationBenchmark.java

```diff
@@ -30,15 +30,25 @@
 import org.apache.tuweni.bytes.Bytes;
 import org.apache.tuweni.bytes.Bytes32;
 import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OperationsPerInvocation;
 import org.openjdk.jmh.annotations.OutputTimeUnit;
 import org.openjdk.jmh.annotations.Param;
 import org.openjdk.jmh.annotations.Scope;
 import org.openjdk.jmh.annotations.Setup;
 import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
 
 @State(Scope.Thread)
-@OutputTimeUnit(value = TimeUnit.MILLISECONDS)
-public class OneMillionCountLeadingZerosOperationBenchmark {
+@OutputTimeUnit(value = TimeUnit.NANOSECONDS)
+@Warmup(iterations = 1)
```

Comment:
```suggestion
@Warmup(iterations = 1, time = 1, timeUnit = TimeUnit.SECONDS)
```

I think we could also add the Measurement explicitly and also reduce default time of 10 to 1.
```
@Measurement(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
```